README.md

```diff
@@ -1,20 +1,23 @@
 <p align="center">
- <img width="250" src="https://i.imgur.com/AlAf04U.png" />
- <br />
-<a href="https://badge.fury.io/js/vue-gtag">
- <img src="https://badge.fury.io/js/vue-gtag.svg" />
-<a/>
-
- <img src="https://img.shields.io/badge/size-2.07kB-brightgreen.svg" />
- 
+  <img width="250" src="https://i.imgur.com/AlAf04U.png">
+  <br>
+  <a href="https://npm.im/vue-gtag">
+    <img src="https://badgen.net/npm/v/vue-gtag">
+  </a>
+  <a href="https://bundlephobia.com/result?p=vue-gtag">
+    <img src="https://badgen.net/bundlephobia/min/vue-gtag">
```

Comment:
i'd use the min gzip, most of the time that's the file size that people look at

`https://badgen.net/bundlephobia/minzip/vue-gtag`

Reply:
Sure, I'll update it.